MapleServer2/Packets/BannerListPacket.cs

```diff
@@ -25,7 +25,7 @@ public static Packet SetBanner(List<Banner> banners)
                 }
                 pWriter.WriteUnicodeString("");
                 pWriter.WriteUnicodeString(banner.ImageUrl);
-                pWriter.WriteEnum(banner.Language);
+                pWriter.WriteInt((int) banner.Language);
```

Comment:
does `pWriter.Write(banner.Language);` works here? since it was an enum.

Reply:
yeah mb